doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -214,6 +214,9 @@ Reference to "Architecture" in this chapter refers to the NFVI Hardware (e.g. ph
 | `req.inf.ntw.14` | Network | The Architecture **must** support capabilities for integrating SDN controllers. |
 | `req.inf.ntw.15` | Network | The Architecture **should** support Service Mesh Interface (SMI). |
 | `req.inf.ntw.16` | Network | The Architecture **should**  support coexistence of multiple connection points in a pod. |
+| `req.inf.ntw.17` | Network | The Architecture **may**  support more than one networking solution.
```

Comment:
Can I suggest splitting this into two requirements as per persistent storage:

`The Architecture must be able to support more than one networking solution.`

`The Architecture must support the ability for an operator to choose whether or not to deploy more than one networking solution.`